<html
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:i18n="http://genshi.edgewall.org/i18n"
  xmlns:py="http://genshi.edgewall.org/"
  xmlns:xi="http://www.w3.org/2001/XInclude"
  py:strip="">

  <xi:include href="../facets.html" />
  <py:def function="page_title">Search - ${g.site_title}</py:def>
  <py:def function="page_heading">Search - ${g.site_title}</py:def>

  <py:match path="primarysidebar">
    
    <li>${facet_sidebar('tags')}</li>
    <li>${facet_sidebar('res_format')}</li>
    <li>${facet_sidebar('groups', label=h.group_name_to_title)}</li>

  </py:match>

  <div py:match="content">
    <xi:include href="search_form.html" />
    ${field_list()}   
    
      <py:if test="c.query_error">
        <p i18n:msg="item_count"><strong>There was an error while searching.</strong> 
            Please try again.</p>
      </py:if>
      <py:if test="request.params">      
        <h4 i18n:msg="item_count"><strong>${c.page.item_count}</strong> datasets found</h4>            
     </py:if>
      <py:if test="c.page.item_count == 0 and request.params">
        <p i18n:msg="">Would you like to <a href="${h.url_for(action='new', id=None)}">create a new dataset?</a></p>
      </py:if>
      ${package_list_from_dict(c.page.items)}
      ${c.page.pager(q=c.q)}

  </div>
  <xi:include href="layout.html" />
</html>


